backend/src/main/java/io/github/butcher/butcher/back/controller/PlayerController.java

```diff
@@ -22,8 +23,8 @@ public PlayerController(PlayerService playerService) {
   }
 
   @PostMapping(value = "/players")
-  public PlayerDTO createPlayer(@RequestBody UIDDTO uiddto) {
-    return playerService.createPlayer(uiddto.getUid());
+  public Mono<PlayerDTO> createPlayer(@RequestBody UIDDTO uiddto) {
+    return Mono.just(playerService.createPlayer(uiddto.getUid()));
```

Comment:
btw. for you devs: this is how you create reactive (non-blocking) request-controllers. because we have netty and webflux (so basically, because we can 😄)